crates/lib/src/config.rs

```diff
@@ -57,16 +58,17 @@ mod tests {
     #[test]
     fn test_load_valid_config() {
         let config_content = r#"
-            [validation]
-            max_allowed_lamports = 1000000000
-            max_signatures = 10
-            allowed_programs = ["program1", "program2"]
-            allowed_tokens = ["token1", "token2"]
-            allowed_spl_paid_tokens = ["token3"]
-            disallowed_accounts = ["account1"]
-
-            [kora]
-            rate_limit = 100
+        [validation]
+        max_allowed_lamports = 1000000000
+        max_signatures = 10
+        allowed_programs = ["program1", "program2"]
+        allowed_tokens = ["token1", "token2"]
+        allowed_spl_paid_tokens = ["token3"]
+        disallowed_accounts = ["account1"]
+        token_symbols = { "So11111111111111111111111111111111111111112" = "SOL", "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v" = "USDC", "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB" = "USDT" }
```

Comment:
You should be able to get the symbol from the token itself instead of relying on manual mapping